spec/presenters/admin/product_presenter/card_spec.rb

```diff
@@ -35,6 +35,7 @@
         expect(props[:alive]).to eq(product.alive?)
         expect(props[:is_adult]).to eq(product.is_adult?)
         expect(props[:is_tiered_membership]).to eq(product.is_tiered_membership?)
+        expect(props[:comments_count]).to eq(product.comments.size)
```

Comment:
We might want to just `expect(props).to eq(...)` if it's feasible so we're forced to update the spec in future

Reply:
Done here: 61e4410. Except for the `cover_placeholder_url` for which I still use a regexp match, the rest now uses exact comparison.